bin/node-pg-migrate

```diff
@@ -221,6 +212,32 @@ if (argv.help || argv._.length === 0) {
   process.exit(1)
 }
 
+/* Load env before accessing process.env */
+const envPath = argv[envPathArg]
+
+try {
+  // Create default dotenv config
+  const dotenvConfig = { silent: true }
+
+  // If the path has been configured, add it to the config, otherwise don't change the default dotenv path
+  if (envPath) {
+    dotenvConfig.path = envPath
+  }
```

Comment:
**suggestion (if-minor):** Line 215 to 225 will never throw an error.  
So should we move them above the `try`?

Reply:
Will "merge" this fix with the comment below